fastify.js

```diff
@@ -97,6 +97,7 @@ function fastify (options) {
   options.requestIdLogLabel = requestIdLogLabel
   options.disableRequestLogging = disableRequestLogging
   options.ajv = ajvOptions
+  options.clientErrorHandler = options.clientErrorHandler || defaultClientErrorHandler
```

Comment:
we should run `node build/build-validation.js` to update the input validation.

Morover the `defaultClientErrorHandler` function should be moved to `defaultInitOptions`

Reply:
`defaultInitOptions` is auto-generated from JSON schema using `ajv`.
`defaultClientErrorHandler` is a function and hence not supported by `ajv`.